push_source.go

```diff
@@ -220,28 +223,23 @@ func (source *Source) createLocale(client *phraseapp.Client, localeFile *LocaleF
 		localeParams.Code = &localeFile.Code
 	}
 
-	localeDetails, err = client.LocaleCreate(source.ProjectID, localeParams)
+	localeDetails, err := client.LocaleCreate(source.ProjectID, localeParams)
 	if err != nil {
 		return nil, err
 	}
 	return localeDetails, nil
 }
 
-func (source *Source) localeShow(client *phraseapp.Client, localeFile *LocaleFile) (*phraseapp.LocaleDetails, error) {
-	identifier := localeIdentifier(source, localeFile)
-	if identifier == "" {
-		return nil, nil
-	}
-
+func (source *Source) getLocaleIfExist(client *phraseapp.Client, identifier string) (*phraseapp.LocaleDetails, error, bool) {
```

Comment:
please move the found to the second parameter, by convention the error is always the last return argument